vcs/git/patch.go

```diff
@@ -19,10 +19,8 @@ func isFileBinaryGit(filename string) (bool, error) {
 		return false, errors.Trace(err)
 	}
 
-	out, err := gitCMD("-C", repoRoot, "diff", "--no-index", "--numstat", "/dev/null", filename)
-	if err != nil {
-	    return false, errors.Trace(err)
-	}
+	// This command is allowed to fail. We only need to grep its output
+	out, _ := gitCMD("-C", repoRoot, "diff", "--no-index", "--numstat", "/dev/null", filename)
```

Comment:
It would be better to ignore only the expected error, if we know what that is.

Reply:
The error code is '1'. I'm guessing that's the generic error code.

Reply:
Ahh 1 means there were differences.